spec/features/admin/reports_spec.rb

```diff
@@ -536,12 +538,12 @@ def xero_invoice_account_invoice_row(adjustment, opts={})
     def xero_invoice_row(sku, description, amount, quantity, tax_type, opts={})
       opts.reverse_merge!({customer_name: 'Customer Name', address1: 'customer l1', city: 'customer city', state: 'Victoria', zipcode: '1234', country: country.name, invoice_number: order1.number, order_number: order1.number, invoice_date: '2015-04-26', due_date: '2015-05-26', account_code: 'food sales'})
 
-      [opts[:customer_name], 'customer@email.com', opts[:address1], '', '', '', opts[:city], opts[:state], opts[:zipcode], opts[:country], opts[:invoice_number], opts[:order_number], opts[:invoice_date], opts[:due_date],
+      [opts[:customer_name], 'customer@email.com', opts[:address1], '', opts[:city], opts[:state], opts[:zipcode], opts[:country], opts[:invoice_number], opts[:order_number], opts[:invoice_date], opts[:due_date],
```

Comment:
I don't think we should alter the spec. Some reports like the Zero report are parsed by other services and they may be very particular about some details.